### PR TITLE
Button Sizing And Fixes

### PR DESCRIPTION
**Pull request by davidenwang**
Created different button sizes through the "size" prop which can be "small", "medium", or "large"

image

Resolves: #46

### CODE REVIEW

**Comment by tko22** [comment]
oh wow good catch! But i think 48px is REALLY big.. maybe ill talk to chloe about making it smaller or maybe we could have a `size` prop for "large", "medium", "small" button - 48px seems to be a big button (when i put it in storybook and compare it to other websites)

**Comment by tko22** [comment]
oh we probably should add these to the button? 
```
text-align: center;
vertical-align: middle;
display: inline-block;
```

**Comment by davidenwang** [comment]
Sounds good to me chief I'll add the styles in and await the updates. Tbh everything in the playground seems bigger than the notion which makes me paranoid lol 😓

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -49,9 +48,13 @@ const ButtonBase = styled.button`
 `;
 
 const Button = ({
-  variant, outline, className, theme: propTheme, ...props
+  variant, outline, className, theme: propTheme, size, ...props
 }) => {
-  const buttonColorPalette = propTheme.colors.variants[variant] || get('colors.variants.primary');
+  const buttonSizingTypography = propTheme.buttons.sizingAndTypography;
+  const sizingAndTypography = buttonSizingTypography[size] || buttonSizingTypography.medium;
```
Comment (tko22) [inline src/components/Button/Button.js:54]: do u know if you could use a variant for this? like https://github.com/hack4impact-uiuc/bridge/blob/master/src/components/Text/Text.js

but with `buttons.sizingAndTypography`

Comment (davidenwang) [reply src/components/Button/Button.js]: yup changed to variant

**Pull request merged**